app/letter_attachment/rest.py

```diff
@@ -0,0 +1,44 @@
+from flask import Blueprint, jsonify, request
+
+from app.dao.templates_dao import dao_get_template_by_id, dao_update_template
+from app.errors import InvalidRequest, register_errors
+from app.letter_attachment.schema import post_create_letter_attachment_schema
+from app.models import LetterAttachment
+from app.schema_validation import validate
+
+letter_attachment_blueprint = Blueprint("letter_attachment", __name__)
+register_errors(letter_attachment_blueprint)
+
+
+@letter_attachment_blueprint.route("/letter_attachment/<uuid:letter_attachment_id>", methods=["GET"])
+def get_letter_attachment(letter_attachment_id):
+    letter_attachment = LetterAttachment.query.get_or_404(letter_attachment_id)
+
+    return jsonify(letter_attachment.serialize())
+
+
+@letter_attachment_blueprint.route("/letter_attachment", methods=["POST"])
+def create_letter_attachment():
+    data = request.get_json()
+
+    validate(data, post_create_letter_attachment_schema)
+
+    if (template := dao_get_template_by_id(data["template_id"])) is None:
+        raise InvalidRequest("Template not found", 404)
+
+    if template.letter_attachment:
+        # TODO: error strings for admin app should ideally be defined in admin. decide what to return from this endpoint
+        raise InvalidRequest("Template already has attachment", 400)
```

Comment:
ah i clearly forgot to action this todo. for now i'll make this `InvalidRequest("template-already-has-attachment")` but in the future I wouldn't mind a specific exception class (and maybe a different json format) to indicate that we expect the string to stay stable so that we can read it elsewhere